Wikipedia/Code/NewArticleTabCoordinator.swift

```diff
@@ -135,25 +138,74 @@ final class NewArticleTabCoordinator: Coordinator {
                         relatedArticles: relatedRecords
                     )
                     becauseVM?.onTapArticle = onTapArticleAction
-
                 }
             }
-
+            
+            let dykVM = WMFNewArticleTabDidYouKnowViewModel(
+                fromSourceDefault: self.fromWikipediaDefault,
+                languageCode:  self.dataStore.languageLinkController.appLanguage?.languageCode,
+                dykLocalizedStrings: WMFNewArticleTabDidYouKnowViewModel.LocalizedStrings.init(
+                dyk: self.dyk,
+                fromSource: self.fromLanguageWikipediaTextFor(languageCode: self.dataStore.languageLinkController.appLanguage?.languageCode),
+                ))
+            
             let viewModel = WMFNewArticleTabViewModel(
                 title: CommonStrings.newTab,
-                becauseYouReadViewModel: becauseVM
+                becauseYouReadViewModel: becauseVM,
+                dykViewModel: dykVM
             )
+
             let vc = WMFNewArticleTabViewController(
                 dataStore: self.dataStore,
                 theme: self.theme,
                 viewModel: viewModel
             )
-            
+
+            self.fetchDYK { facts in
+                DispatchQueue.main.async {
+                    dykVM.facts = facts?.map { $0.html }
+                }
+            }
+
             self.navigationController.pushViewController(vc, animated: true)
         }
+
         return true
     }
+    
+    // MARK: - DYK
+    
+    let fromLanguageWikipedia = WMFLocalizedString("new-article-tab-from-language-wikipedia", value: "from %1$@ Wikipedia", comment: "Text displayed as Wikipedia source on new tab. %1$@ will be replaced with the language.")
```

Comment:
We can check with design on this, but the "from language Wikipedia" format doesn't work in all languages.
Example from Portuguese: 
"da Wikipédia em Português" which roughly translates as "From Wikipedia in Portuguese", and not "From Portuguese Wikipedia".
So I'm not sure how that would be put togheter in other languages.
It might just work out in translate wiki, but I thought about flagging it.